rfcs/protocols/asset-transfer/generic.md

```diff
@@ -6,13 +6,14 @@
 # Asset Transfer Protocol
 
 - RFC: 02-008
-- Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
+- Authors: Krishnasuri Narayanam, Sandeep Nishad, Venkatraman Ramakrishna, Dhinakaran Vinayagamurthy
```

Comment:
In `corda-htlc.md`, you can put yourself at the head of the authors' list. but the first protocol spec and implementation (in Fabric) were led by me, so I think I should be at the head of this one.